apps/api/src/report/constants/milestone-id.ts

```diff
@@ -0,0 +1,58 @@
+import { STATUS } from '@ien/common';
+
+export const STATUS_ID: Record<STATUS, string> = {
+  [STATUS.APPLIED_TO_BCCNM]: '8a9b0d13-f5d7-4be3-8d38-11e5459f9e9a',
+  [STATUS.APPLIED_TO_NNAS]: 'e68f902c-8440-4a9c-a05f-2765301de800',
```

Comment:
can we use getStatusMap() instead of this?  It returns the statuses in the same format `'Candidate Withdrew from Competition': '3fd4f2b0-5151-d7c8-6bbc-3a0601b5e1b0'`

Reply:
Good catch! I feel trapped in a labyrinth.